password manager

mercurrie opened the pull request:
password manager feature

mercurrie committed:
password manager

complete password manager

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,78 @@
+                password_entry.delete(data["START_INDEX"], END)
+
+
+# window
```
You generally want to run any code below this:

if __name__ == '__main__':

mercurrie replied:
corrected

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,78 @@
+import json
+from tkinter import *
+from tkinter import messagebox
+data_file = open("data.json")
```
One line

mercurrie replied:
unfortunately it won't let me include messagebox on the same line as * because it is a module.

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,78 @@
+window.title(data["WINDOW_TITLE"])
+window.config(padx=data["WINDOW_PAD_X"], pady=data["WINDOW_PAD_Y"])
+
+# image
```
A lot of this logic could be encapsulated within functions

mercurrie replied:
corrected. Due to the following code below calling the save function when clicking the add button. Without being able to pass parameters for the website_entry, email_entry, and password_entry variable, I had to create a Class to initialize these variables to allow them to be shared across functions

`add_button = Button(text=data["ADD_BUTTON_TEXT"], width=data["ADD_BUTTON_WIDTH"], command=### save)`

michaelhousman reviewed (changes requested):
Looks good.  Added a few comments.

mercurrie committed:
feature revision

included __name___ == '__main__'
turned features into a class to effectively use functions to split up code.

All functions of tkinter are imported
Note messagebox is a module and has to be imported separately
